app/src/main/res/values/dimens.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <dimen name="spacing">20dp</dimen>
```

Comment:
WARNING: [UnusedResources] The resource `R.dimen.spacing` appears to be unused
 Unused resources make applications larger and slow down builds.